source/spark_sql_migrations/spark_sql_migrations/current_state/__init__.py

```diff
@@ -0,0 +1,4 @@
+"""
+By having a __init__.py in this root directory,
+we can use the test explorer in VS Code.
+"""
```

Comment:
This content occurs everywhere. But I'm not sure we really use VS Code anymore. But what's important is that the modules of the folder is not included in the wheel if the `__init__.py` does not exist. So perhaps just remove this content from all init files?